device/bgp/bgp.yaml

```diff
@@ -82,3 +82,15 @@ components:
           type: array
           items:
             $ref: './bgpv6routerange.yaml#/components/schemas/Bgp.V6RouteRange'
+        v4_srte_policies:
+          description: >-
+            Segment Routing (SR) Policy for IPv4 Address family indicator (AFI).
+          type: array
+          items:
+            $ref: './bgpv4srtepolicy.yaml#/components/schemas/Bgp.V4Policy'
+        v6_srte_policies:
+          description: >-
+            Segment Routing (SR) Policy for IPv6 Address family indicator (AFI).
```

Comment:
Make the first alphabet capital for “family” and “indicator”. It should be “IPv6 Address Family Indicator (AFI)”.